backup/barman-montagu

```diff
@@ -20,8 +20,10 @@ Options:
   --slot=<slot>             Replication slot to use on the db
   --volume_data=<volume>    Volume name or path for data [default: barman_data]
   --volume_logs=<volume>    Volume name or path for logs [default: barman_logs]
-  --volume_recover=<volume> Volume name or path for recovery [default: barman_recover]
-  --volume_nightly<volume>  Volume name or path for nightly recover [default: montagu_db_volume]
+  --volume_recover=<volume> Volume name or path for recovery
+                            [default: barman_recover]
+  --volume_nightly<volume>  Volume name or path for nightly recover
+                            [default: montagu_db_volume]
```

Comment:
Turns out the long lines here were breaking docopt(!) - it wasn't parsing the default values.

Reply:
I didn't know you could break across multiple lines like that - that's really good to know!